libs/wlgen/wlgen/rta.py

```diff
@@ -101,6 +101,39 @@ def __init__(self,
         # Setup RTA callbacks
         self.setCallback('postrun', self.__postrun)
 
+    @staticmethod
+    def __setGovernorsPerformance(target):
+        if 'cpufreq' not in target.modules:
+            log = logging.getLogger('RTApp')
+            log.warning('cpufreq module not loaded,'
+                        ' skipping setting frequency to max')
```

Comment:
can we keep this space at the end of the previous sentence?

Reply:
I'd prefer no line break for user-exposed message for the same reason as in the kernel coding style: if you want to know where a message comes from, you grep for it, and splitting the message will defeat simple grep commands.